### ISSUE TITLE
[question] How to run this on AWS ECS?

### ISSUE THREAD

**Issue by lewisbarber** [gten/docker-zookeeper-cluster#1]
I don't see a way to run this on AWS Container Service in a single cluster (with, for example, 3 EC2 instances). This is because each cluster has to use a single Container definition (docker run...), and I need to pass a variable to docker run that is different for each EC2 instance in my cluster. In other words, I think I need 3x AWS clusters, each with a single EC2 instance and slightly different Container definition setup.

I don't suppose you've tried this? Or come up with a solution?

**Comment by jeygeethan** [comment]
What are you trying to do is run a zookeeper cluster on AWS with 3 EC2 instances, is that right?

**Comment by lewisbarber** [comment]
Yes, but using Amazon Container Service to deploy the Docker image from an Amazon Docker Repository.

If you're not familiar with this, then the question may not make a lot of sense! I've forked your repo and made some small changes, to support this deployment approach. You can see here: https://github.com/bettle-app/docker-zookeeper-cluster

Let me know if you'd be happy to merge these changes, and I'll update the read me and send a pull request.

**Comment by jeygeethan** [comment]
I am ok with the pull request. Can you send one soon?
